rotkehlchen/chain/ethereum/modules/eth2/beacon.py

```diff
@@ -113,7 +126,10 @@ def query_chunked(
         chunks = calculate_query_chunks(indices_or_pubkeys, chunk_size=chunk_size)
         data = []
         for chunk in chunks:
-            data.extend(self.query(querystr.format(chunk=','.join(str(x) for x in chunk))))
+            data.extend(self.query(
+                endpoint=endpoint,
+                data={'ids': [str(x) for x in chunk]},
+            ))
```

Comment:
checking where it is used `calculate_query_chunks` seems to always be used as string so:

1. Confirm that we always do `str(x) for x in calculate_query_chunks`
2. make `calculate_query_chunks` return a list of str directly. In fact inside `calculate_query_chunks` we use `list()` so we are iterating always twice